### PR TITLE
Add `_set_fitted_parameters` to `FloatFormatter`

### LINKED ISSUE #829: Add `_set_fitted_parameters` method to `FloatFormatter`
### Problem Description
We'd like to be able to get a transformer into a fitted state using known parameters instead of existing data. This would allow us to set up a transformer to be able to reverse transform without first having to fit it on real data.

### Expected behavior
Add the method `_set_fitted_parameters` to the `FloatFormatter`. When called, the provided arguments should be set on the transformer to get it into a 'fitted' state so that it can be used to reverse transform. After being called, a user should be able to call `reverse_transform` and have it work as expected.

def _set_fitted_parameters(self, output_columns, min_max_values, null_transformer, rounding_digits, dtype='float64')
- Args:
  - column_names [str]: The name of the column to use for the transformer. Used to set `self.columns` and `self.output_columns`.
  - min_max_values [tuple or None]: `None` or a tuple containing the (min, max) values for the transformer. Should be used to set `self._min_value` and `self._max_value`. If `self.enfore_min_max_values` is `True`, it cannot be `None`.
  - null_transformer [a `NullTransformer` instance]: A _fitted_ null transformer instance that can be used to generate null values for the column.
  - rounding_digits [`None` or int]: The number of digits to round to. Should be used to set `self._rounding_digits`. If `None` then `self.learn_rounding_scheme` must be `False`.
  - dtype [str]: The pandas dtype the reversed data should be converted to. Should be used to set `self._dtype`.

This method should not return anything. It should validate the parameters that are passed in, especially that the null transformer is compatible.

### PR DESCRIPTION
**Pull request by lajohn4747**
resolves #829 
CU-86b0pvkjj

### CODE REVIEW

**Comment by sdv-team** [comment]
Task linked: [CU-86b0pvkjj RDT - Add `set_fitted_parameters` method to `FloatFormatter` #829](https://app.clickup.com/t/86b0pvkjj)

File: rdt/transformers/numerical.py [hunk 1]
```diff
@@ -198,6 +198,38 @@ def _reverse_transform(self, data):
...
+                The number of digits to round to.
+            dtype (str):
+                The pandas dtype the reversed data should be converted to
+        """
+        self.null_transformer = null_transformer
+        self.columns = [column_name]
+        self.output_columns = [column_name]
+        if min_max_values:
+            self._min_value = min(min_max_values)
+            self._max_value = max(min_max_values)
```
Comment (frances-h) [inline rdt/transformers/numerical.py:228]: If `self.enforce_min_max_values` is `True` we should error if these aren't provided

**Review by frances-h — APPROVED** [review]

**Review by fealho — COMMENTED** [review]
Same as for your other PR, could you add an integration test?

**Review by fealho — APPROVED** [review]

File: rdt/transformers/numerical.py [hunk 2]
```diff
@@ -198,6 +199,44 @@ def _reverse_transform(self, data):
...
+                The name of the column to use for the transformer.
+            null_transformer (NullTransformer):
+                A fitted null transformer instance that can be used to generate
+                null values for the column.
+            min_max_values (Tuple(float) or None):
+                None or a tuple containing the (min, max) values for the transformer.
+            rounding_digits (int or None):
+                The number of digits to round to.
+            dtype (str):
+                The pandas dtype the reversed data should be converted to
```
Comment (fealho) [inline rdt/transformers/numerical.py:222]: Missing full stop.

**Review by frances-h — APPROVED** [review]

**Pull request merged**